api/test/wfl/tools/workloads.clj

```diff
@@ -23,6 +23,10 @@
 (def email
   (delay (:email (gcs/userinfo {:headers (auth/get-auth-header)}))))
 
+(def watchers
+  [["EmailAddress" "hornet@broadinstitute.org"]
```

Comment:
I don't think you can change this without making a breaking change with what's in prod. I'd keep `watchers` to be an email address or a `["SlackChannel" "{channel-id}"]` pair.

```
Watcher -> EmailAddress | [ "SlackChannel" ChannelId ] 
```

Reply:
My concern is that we will never have a chance to standardize the emails into tagged pairs, especially now we are not sending anything to any email watchers in prod. Do you think the "breaking change" can be done with a Postgres liquibase migration? e.g. use a cast or something similar to migrate all existing emails string records in prod to tagged pairs. It may challenge my psql skill though.

Reply:
Yes, that's the point - we won't and don't have to. It's in the dev and prod database as this. If you can do this in  db migration, fine. I think making a user type `["EmailAddress"` before every email address is a pain.

Reply:
I see, all make sense to me, will probably make it `Watcher -> EmailAddress | [ "SlackChannel" ChannelId ] `, as it leaves minimal work for myself to update. (I was just having OCD seeing them differ (string and a pair) 🤦 )

Reply:
Howzabout?
Watcher -> ["SlackChannel" ChannelId] | ["EmailAddress" EmailAddress] | EmailAddress